lib/hardware_services/network_drivers/HilNetworkDriver.py

```diff
@@ -14,19 +14,74 @@
 from subprocess import call
 from subprocess import check_call
 
-from hardware_services.network_service import NetworkService
+sys.path.append(os.path.join(os.path.dirname(__file__), "..", "util"))
 
-# added for EC528 HIL-QUADS integration project
-hil_url = 'http://127.0.0.1:5000'
+import hilapi
```

Comment:
And change this to `import lib.hardward_services.utils.hilapi`, where we add a __init__.py module to the `utils` directory, won't we avoid appending this to the path?